source/MaterialXRuntime/RtFileIo.h

```diff
@@ -113,6 +113,9 @@ class RtExportOptions : public RtWriteOptions
 
     // Libraries to load
     DocumentPtr libraries;
+
+    // Predicate to use to skip flattening
+    SkipFlattenPredicate skipFlattening;
```

Comment:
If this is a `FilePathPredicate` then the member can be called `filePathPredicate` which seems to be the convention.
Might want to group this with `flattenFilenames` and mention it will only trigger if the the option is enabled.